java/src/main/java/scenarios/crewcreation/CharacterCreation.java

```diff
@@ -0,0 +1,250 @@
+package scenarios.crewcreation;
+
+import crew.Captain;
+import system.gameplay.engine.io.DataPrinter;
+import system.gameplay.engine.io.SystemInputScannerSingleton;
+import system.gameplay.roleplay.DiceRoller;
+
+import java.util.*;
+
+public final class CharacterCreation {
+
+    private static final Scanner scanner = SystemInputScannerSingleton.getInstance();
+
+    // TODO: Move these constants to an enum
+    private static final String PHYSICAL = "Physical";
+    private static final String AGILITY = "Agility";
+    private static final String BLASTER = "Blaster";
+    private static final String PILOTING = "Piloting";
+    private static final String REPAIR = "Repair";
+    private static final String CHARISMA = "Charisma";
+
+    public static Captain runCharacterCreation() {
+        System.out.println("First things first, what is your name...?");
+        String name = scanner.nextLine();
+
+        System.out.println(
+                name + ", eh? Not the name I was expecting for a star ship captain, if I'm being honest with you..."
+        );
+
+        // TODO: Add a bit of flavor text describing each ability score, move into seperate describeAbilityScores method
+        System.out.println("Anywho, Ability Scores are what really show what someone's made of!");
+
+        System.out.println("Well then, what might your ability scores be?");
+        System.out.println("Press any key to roll the dice and determine your potential attributes...");
+        scanner.nextLine();
+
+        List<Integer> unassignedAbilityScores = DiceRoller.rollAbilityScores();
+        // TODO: Implement functionality of the rollAbilityScore method
+
+        int meanScore = (int) unassignedAbilityScores.stream().mapToInt(i -> i).average().orElse(0);
+
+        System.out.println("Looks like these are your ability scores: " + unassignedAbilityScores);
+
```

Comment:
I had left this big if else block in this method back when we had this code in the `Main` class because I didn't want to fill the `Main` up with too many methods. Now that we have CharacterCreation in it's own class, we can move this block into it's own method, something like:
```java
private static void printMeanScoreFlavorText(int meanScore) {

}
```

Since `meanScore` will not be used anywhere outside of the function, we can create it in the same line that we call the new method in. This way, `meanScore` will only exist within the scope of this new method (Separation of Concerns, once again.) To do this, simply pass in the code used to get the `meanScore` as an argument when calling the new method.
```java
printMeanScoreFlavorText((int) unassignedAbilityScores.stream().mapToInt(i -> i).average().orElse(0));
```